src/event_gate_lambda.py

```
@@ -84,55 +84,49 @@
-        "headers": {"Content-Type": "application/json"},
-        "body": json.dumps(TOPICS[topicName])
-    }
+
+    return {"statusCode": 200, "headers": {"Content-Type": "application/json"}, "body": json.dumps(TOPICS[topicName])}
+
 
 def post_topic_message(topicName, topicMessage, tokenEncoded):
     logger.debug(f"Handling POST {topicName}")
     try:
         token = jwt.decode(tokenEncoded, TOKEN_PUBLIC_KEY, algorithms=["RS256"])
     except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)